plugins/status/plugin_test.go

```diff
@@ -28,36 +28,21 @@ import (
 )
 
 const resp = `plugin: http, status: 200
+plugin: jobs, status: 200
 plugin: rpc not found`
 
-func TestStatusHttp(t *testing.T) {
+const pluginQuery = `plugin=http&plugin=jobs&plugin=rpc`
+
+func TestStatus(t *testing.T) {
 	cont := endure.New(slog.LevelDebug)
 
 	cfg := &config.Plugin{
-		Version: "2.9.0",
-		Path:    "configs/.rr-status-init.yaml",
+		Version: "3",
```

Comment:
There is no version `3` exists. I guess you wanted to use here `v2023.1.x` ?

Reply:
Fixed, the concepts of the config version and the plugin version are confused in my head

Reply:
Just think about docker and `docker-compose.yaml`. Docker version is 21.xx, but `docker-compose` version is `2.18.x`, aaand `docker-compose.yaml` version is `3.8`. They are completely independent, as are the RR version and the `.rr.yaml` configuration version.
RR version 2023 supports configurations from version `3`, but not older `2.7`. This is done to have backward compatibility when we update the configuration. For example: RR version 2023.3 for example might have support for updated configuration `3.1`.

Reply:
In the `config.Plugin`, `Version` field is the RR version, not the plugin. When it compiled in bundle it sets automatically with during the binary linking stage. For the tests we just manually set it.

Reply:
Realization of this feature might optimize e2e RR configs. As i see now we should copy-paste many rows https://github.com/roadrunner-server/roadrunner/issues/935#event-5900619897

Reply:
Yeah, I expected this feature from Viper, like, 2 yrs ago, but now I'm thinking to implement this feature by myself (or with the community help). That's not so hard to do.